packages/web-components/src/components/masthead/__stories__/masthead.stories.ts

```diff
@@ -17,7 +17,12 @@ import '../masthead-container';
 import { L1_CTA_TYPES } from '../defs';
 import styles from './masthead.stories.scss';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { mastheadL0Data, mastheadL1Data, mastheadLogoData } from './links';
+import {
+  mastheadL0Data,
+  mastheadL1Data,
+  mastheadL1EmptyMenuItemsData,
```

Comment:
This doesn't appear to be an export from `./links.ts`

Reply:
Ohh boo, I forgot to commit the changes to that. Coming right up.